tigerpath/majors_and_certificates/majors/SPO_2018.json

```
@@ -0,0 +1,359 @@
+                  "course_list": [
+                    "ANT 3*",
+                    "ANT 4*",
+                    "ANT 5*",
+                    "ARC 3*",
+                    "ARC 4*",
+                    "ARC 5*",
+                    "COM 3*",
+                    "COM 4*",
+                    "COM 5*",
+                    "ECS 3*",
+                    "ECS 4*",
+                    "ECS 5*",
+                    "ENV 3*",
+                    "ENV 4*",
+                    "ENV 5*",
+                    "GHP 3*",
+                    "GHP 4*",
+                    "GHP 5*",
+                    "HIS 3*",
+                    "HIS 4*",
+                    "HIS 5*",
+                    "HUM 3*",
+                    "HUM 4*",
+                    "HUM 5*",
+                    "LAO 3*",
+                    "LAO 4*",
+                    "LAO 5*",
+                    "LAS 3*",
+                    "LAS 4*",
+                    "LAS 5*",
+                    "POL 3*",
+                    "POL 4*",
+                    "POL 5*",
+                    "SOC 3*",
+                    "SOC 4*",
+                    "SOC 5*",
+                    "URB 3*",
+                    "URB 4*",
+                    "URB 5*",
+                    "WWS 3*",
+                    "WWS 4*",
+                    "WWS 5*"
+                  ]
+                },
+                {
+                  "name": "Creative Arts",
+                  "max_counted": 1,
+                  "min_needed": 3,
+                  "explanation": "Concentration in Spanish or Portuguese with the creative arts (e.g. creative writing, theater, visual arts, translation). Requires a combination of five upper-division courses in Spanish or Portuguese and three upper-division courses in the creative arts. One of the five upper-division courses in Spanish or Portuguese may be taken in English, in which case all written work must be completed in the target language.",
+                  "course_list": [
+                    "CRW 3*",
+                    "CRW 4*",
+                    "CRW 5*",
+                    "DAN 3*",
+                    "DAN 4*",
+                    "DAN 5*",
+                    "THR 3*",
+                    "THR 4*",
+                    "THR 5*",
+                    "TRA 3*",
+                    "TRA 4*",
+                    "TRA 5*",
+                    "VIS 3*",
+                    "VIS 4*",
+                    "VIS 5*"
+                  ]
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "name": "Portuguese",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": null,
+          "req_list": [
+            {
+              "name": "Prerequisites",
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "All Portuguese concentrators are required to take at least one 200-level POR course.",
+              "course_list": [

```

See my comment on Spanish above.
```suggestion
              "double_counting_allowed": true,
              "course_list": [
```